subsys/fw_info/Kconfig

```diff
@@ -7,7 +7,8 @@
 menuconfig FW_INFO
 	bool
 	prompt "Firmware Metadata"
-	imply NRFX_NVMC
+	imply NRFX_NVMC if (DT_HAS_NORDIC_NRF51_FLASH_CONTROLLER_ENABLED || DT_HAS_NORDIC_NRF52_FLASH_CONTROLLER_ENABLED || DT_HAS_NORDIC_NRF53_FLASH_CONTROLLER_ENABLED || DT_HAS_NORDIC_NRF91_FLASH_CONTROLLER_ENABLED)
+	imply NRFX_RRAMC if DT_HAS_NORDIC_RRAM_CONTROLLER_ENABLED
```

Comment:
When building app with secure boot still are warnings with NRFX_NVMC.
Could be good to also remove `CONFIG_NRFX_NVMC=y` from:
samples.bootloader.prj.conf, prj_minimal.conf

and update: subsys/bootloader/bl_validation/Kconfig

Reply:
I can build and flash a board, but the board boots only to a massage `Trying to get Firmware version` and it is stuck (no expected "Hello world"). 
Output from UART:
```
*** Booting nRF Connect SDK v2.7.99-32aa981f20e8 ***
*** Using Zephyr OS v3.6.99-96aa87c2aae4 ***
Attempting to boot slot 0.
Attempting to boot from address 0xb6000.
I: Trying to get Firmware version
```
Steps to reproduce:
```
west build $ZEPHYR_BASE/samples/hello_world --pristine -b nrf54l15pdk/nrf54l15/cpuapp -d build-54l-hellonsib -- -DSB_CONFIG_SECURE_BOOT_APPCORE=y -Db0_CONFIG_NRFX_RRAMC=y
west flash --skip-rebuild -d build-54l-hellonsib --recover
minicom -D /dev/ttyACM1
```
Tested on nRF54L15-PDK 7.0.0